firebase-storage-app/firebase-storage-app.gradle

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+apply plugin: "com.android.application"
```

Comment:
You should also apply the following plugin as connected-check has been disabled in CI
```
apply plugin: com.google.firebase.gradle.plugins.ci.device.FirebaseTestLabPlugin
```

Reply:
As discussed offline, there are no Test Lab tests in this app.